app/models/grda_warehouse/auth_policies/user_acl_context.rb

```diff
@@ -32,6 +32,10 @@ def initialize(user)
     permissions_for_collection_ids(collection_ids)
   end
 
+  def system_permissions
+    permissions_for_collection_ids(system_collection_ids(:data_sources))
+  end
```

Comment:
Can you explain what's going on here?  Why are these called `system_permissions` but only referencing the system collection for data sources?  (Maybe I'm missing something.)  It feels like the intention is to somehow target permissions that don't need to reference a collection.